docs/release-notes/2.25.0.md

```diff
@@ -0,0 +1,88 @@
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+Highlights. 
+** (Technology Preview ) Ability to connect your Percona Monitoring and Management instances to your Percona Portal to get the latest intelligence from Percona and have all your accounts and relations with Percona centralized. READ_MORE
+
+
+
+** From this release, PMM will let users who add PostgreSQL Servers be monitored by PMM to specify database name for monitoring. Previous versions have the default `postgres` database, which wasn't always convenient for the users. 
+
+
+** This Release of PMM is supported and compatible with the recent release of the PostgreSQL extension - pg_stat_monitor v 1.0.0. You can read more about its features and advantages in this blog post (LINK to pg_stat_monitor release announcement or Userguide)
+
+** PMM 2.25 also be compatible and working fine with the latest Percona Disctibutions for PostgreSQL released today (LINK), which includes the newest version of pg_stat_monitor extension.  
+
+** We changed the way how the dashboards, plugins, and data sources are added into PMM. We now use the native Grafana provisioning mechanism in PMM. This change will lead to a faster and more reliable update process for our users from one version of PMM to another. With this update, we'll preserve all custom changes you might already do with plugins and dashboards, but we'll encourage you to be back up them before an upgrade and recheck after, just in case. 
+
+** This version of PMM also will let you change the time zone on dashboards. It will also preserve this selection while you navigate over Dashboards. If you need to change this setting permanently for your account - you need to change it in your preferences by the URL: https://YOUR_PMM_SERVER/graph/profile   
+
+
+** (Technical Preview) DBaaS: PMM will now let you deploy and update your PXC clusters created with DBaaS functionality to the version of Percona Distribution for MySQL Operator 1.9.0 to deliver the latest fixes in this version.
+
+
+** For users who use PMM in automation, we added a pmm-client docker container to be started as a sidecar container and not fail if the connection to DB  is not available. Please see full details in 
+https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/client/index.html#install
+
+
+** With the support of new products and new versions of already supported products, we also removed old software. Since this release, we are removing support of Ubuntu 16.04  in PMM. Please see more details in  
+https://www.percona.com/blog/eol-pmmv1-mongo40-ubuntu1604
+
+
+
+
+ 
+
+## New Features
+
+- [PMM-9050](https://jira.percona.com/browse/PMM-9050): Connect PMM Server to Percona Portal for additional account info in PMM and value added content
```

Comment:
Better to use Percona Platform, or Percona Platform Portal, instead of Percona Portal.

Reply:
@Hlmn-Rmr go ahead and correct ticket titles. I am going to run jira-bot and merge ticket changes couple of times in a next couple of days.